UserAdminBundle/Form/Type/UserType.php

```diff
@@ -13,13 +14,21 @@
 class UserType extends AbstractType
 {
     protected $class;
+    protected $languages;
 
     /**
      * @param string $class
      */
-    public function __construct($class)
+    public function __construct($class, Twig_Environment $twig)
```

Comment:
In this file `OpenOrchestraBaseExtension` The languages 'fr' and 'en' are always set.

In the compiler pass, we should suppress the default values I think